source/limitations.txt

```diff
@@ -0,0 +1,96 @@
+.. _entity-framework-limitations:
+
+===========
+Limitations
+===========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta:: 
+   :keywords: .NET, support
+
+Overview
+--------
+
+On this page, you can find a list of MongoDB and {+framework+} features that
+are not supported by the {+provider-long+}. Because the provider is in active
+development, some features listed on this page might be considered for future
+releases based on customer demand.
+
+MongoDB Features
+----------------
+
+The following sections discuss MongoDB features that are not supported by the {+provider-short+}.
+
+Time Series
+~~~~~~~~~~~
+
+The {+provider-short+} does not support time series data or time series
+collections. These are MongoDB-exclusive features that are not available in {+framework-core+}.
```

Comment:
Q: the second sentences of all of these sections are similar. could we omit all of them? Or omit the first sentence and rephrase the second sentence to something like:

Time series data and time series collections are not supported in {+framework-core+}.

Reply:
I think this is an unfortunate side effect of similar names. The first sentence is saying that the Entity Framework _provider_ (ours) doesn't support these features. The second is clarifying that they're not supported because they are MongoDB features that are not available in Entity Framework (Microsoft's).